src/main/resources/schema/rpa-json-schema.json

```diff
@@ -517,7 +517,7 @@
         "string",
         "null"
       ],
-      "pattern": "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$|^$",
+      "pattern": "^[A-Za-z0-9._'%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$|^$",
```

Comment:
Although technically possible, should we allow such characters in the email address. This may cause issues in downstream system that will feed data from Civil (RPA). Are we sure this is the intended email address and HMCTS are ok to allow such characters in email?